app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,24 @@
+
+<%= stylesheet_link_tag('register-new')%>
+
+<!-- app/views/devise/registrations/new.html.erb -->
+<% if user_signed_in? %>
+<div class="log_in">
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <%= f.email_field :email, autofocus: true, autocomplete: 'email', placeholder: 'username', class: "form_input"  %>
+  <%= f.password_field :password, autocomplete: 'current-password', placeholder: 'password', class: "form_input" %>
+  <%= f.submit 'Log in', class: "log_submit" %>
+  <% end %>
+  <p>Don't have an account? <%= link_to 'Sign up', new_user_registration_path %></p>
+  </div>
+<% else %>
+  <div class="sign-up">
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+     <%= f.email_field :email, autofocus: true, autocomplete: 'email', placeholder: ' youremail@email.com', class: "form-pass"  %>
+  <%= f.password_field :password, autocomplete: 'new-password', placeholder: 'new password', class: "form-pass"  %>
+  <%= f.password_field :password_confirmation, autocomplete: 'new-password', placeholder: 'new password', class: "form-pass"  %>
+  <%= f.submit 'Sign up', class: "submit_sign_up" %>
+  <% end %>
+  <p class="p-account">Already have an account? <%= link_to 'Log in', new_user_session_path %></p>
+  </div>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure the views (login and signup pages) look identical to the provided wireframe

|Your app|Provided wireframe|
|-|-|
|![Screenshot_20230708_111224](https://github.com/TaiwoEnoch/Social_Media_App/assets/25227022/3b4c113e-d738-4478-ae97-b2fb8c749941)|![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)|